docs/platforms/javascript/common/tracing/instrumentation/custom-instrumentation/index.mdx

```diff
@@ -101,6 +102,20 @@ To add spans that aren't active, you can create independent spans. This is usefu
 
 <PlatformContent includePath="performance/start-inactive-span" />
 
+## Starting Spans as Children of a Specific Span
```

Comment:
Are we guarding this paragraph for node guides? AFAIK this won't work in browser SDK where we have a flat span hierarchy, right?

Reply:
hmm, good point, I mean it will kind of work, if the span is a root span for example. I would leave this as is for now, we can maybe clarify this even further if needed!

Reply:
ok sounds good to me!